chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -160,7 +160,7 @@ spec:
               subPath: airflow_local_settings.py
               readOnly: true
 {{- end }}
-{{- if .Values.dags.persistence.enabled }}
+{{- if or .Values.dags.gitSync.enabled .Values.dags.persistence.enabled }}
```

Comment:
Can you please add a test in https://github.com/apache/airflow/blob/master/chart/tests/test_git_sync_scheduler.py